### PR TITLE
Refactor: change "Add Form" button link and styles

### PR DESCRIPTION
**Pull request by glaubersilva**
Resolves [GIVE-999]

## Description

This PR makes the "Add Form" button from the form list view create a new form using the visual form builder. It also updates the styles to keep the consistency with the WordPress secondary and primary buttons.

## Affects

The "Add Form" button from the form list view.

## Visuals

https://www.loom.com/share/caff9f3671eb4defab5b470e391ae55e?sid=e9fee7e0-7817-4af9-ae05-1d9228918d74

## Testing Instructions

1. Go to `Give > All Forms` and make sure the new styles are applied to the buttons;
2. Go to `User > Profile` and change the admin color scheme; 
3. Check if the new color schema got applied to the buttons;
4. Click on the "Add Form" button and make sure it's creating new forms using the Visual Form Builder.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

[GIVE-999]: https://stellarwp.atlassian.net/browse/GIVE-999?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jdghinson — APPROVED** [review]
All good!

**Comment by pauloiankoski** [comment]
@jdghinson Currently, the buttons are being styled slightly different from WP native buttons (32px vs 30px height and 500 vs 400 font-weight). Was that intentional?

CleanShot 2024-07-29 at 13 02 20
_Native on top, modified on bottom_

**Comment by jdghinson** [comment]
@pauloiankoski Yes, the native looks smaller.

**Review by pauloiankoski — CHANGES_REQUESTED** [review]

File: src/Views/Components/ListTable/ListTablePage/ListTablePage.module.scss [hunk 1]
```diff
@@ -120,6 +120,16 @@
...
 
+.buttonSecondary {
+    font-weight: 500;
+    height: 32px;
+}
+
+.buttonPrimary {
+    height: 32px;
+    line-height: 2.4 !important;
+}
```
[the remark this answers is not in the document]
Comment (glaubersilva) [reply src/Views/Components/ListTable/ListTablePage/ListTablePage.module.scss]: Resolved in 630963f8535f09e282e8537cdd7951f08b3514f3

**Comment by glaubersilva** [comment]
@pauloiankoski This is ready for re-review.

**Review by pauloiankoski — APPROVED** [review]
👍🏻

**Review by rickalday — APPROVED** [review]
Passed manual QA tests.

**Pull request merged**